src/js/components/Chart/doc.js

```diff
@@ -186,7 +186,11 @@ export const doc = Chart => {
           label: PropTypes.string, // for accessibility of bars and points
           onClick: PropTypes.func,
           onHover: PropTypes.func,
-          opacity: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+          opacity: PropTypes.oneOfType([
+            PropTypes.string,
+            PropTypes.number,
+            PropTypes.bool,
```

Comment:
Does the TS file needs attention?

Reply:
Nope, the typescript definition already includes boolean as an option for both opacity and values.opacity `opacity?: 'weak' | 'medium' | 'strong' | boolean | number;`